ui-components/menuitems.md

```diff
@@ -0,0 +1,124 @@
+---
+layout: default
+title: Custom menu items
+title_nav: Custom menu items
+description: This section demonstrates different types of menu items.
+keywords: menu menuitem menuitems
+---
+
+## Use Cases
+
+* Create a shortcut for an action or a series of actions that the user repeatedly does.
+* Create a button for custom behavior.
+
+## How to create custom menu items
+
+The methods for adding custom menu items are in the UI Registry part of the editor API editor.ui.registry. The API has three methods for adding menu items:
+
+* `editor.ui.registry.addMenuItem`(identifier, configuration)
+* `editor.ui.registry.addToggleMenuItem`(identifier, configuration)
```

Comment:
Let's order these alphabetically too? Might as well be consistent.

Reply:
Agreed! I completely missed this one, so thanks for the catch :)

Reply:
Fixed :smiley: